### ISSUE TITLE
Retopo Tools

### ISSUE THREAD

**Issue by parsec3d** [umiyuki/Artstage-Issues#56]
Retopo Tools 
 
 Project geometry
 Stick on top of underlaying layer.

**Comment by umiyuki** [comment]
I need more detailed information.
Need something like Blender's Retopo tool?

Re-topology is necessary to arrange the topology of meshes created by voxel sculpting, for example.
Artstage creates a high resolution mesh from low poly with OpenSubdiv. I don't think there is any need for retopology in this workflow.
In your scenario, do you want to retopology models created with other software, in Artstage?

In Blender it is necessary to operate with the mouse so it is necessary to adsorb the vertex to the high resolution mesh. but in VR it is seem easy to create a face by aligning the position with the high resolution mesh without adsorbing the vertex.

**Comment by parsec3d** [comment]
You are right it is easier, but snapping to geometry on a non active layer would be cool to have..
Yes would be to rework models made elsewhere.

**Comment by MarkMironov** [comment]
It seems to me that it’s already easy to do retopology. Another thing is whether the program will open millions of polygons of the original mesh. I do not think so..
Light models can already be imported.
